Add CFP reminder post (#2501)
writethedocs/www: docs/conf/portland/2026/news/cfp-reminder.md

---
template: {{year}}/generic.html
og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
---

```{eval-rst}

> .. post:: Jan 3, 2026

Reviewer: ```suggestion
.. post:: Jan 7, 2026
```

   :tags: {{shortcode}}-{{year}}
```

# Reminders: Call for Proposals, Tickets, Opportunity Grants, and Volunteering

Happy New Year, fellow documentarians!

As we enter {{year}}, we're looking forward to Write the Docs {{city}} and sending some reminders to kick the year off. We hope you're planning to join us for the {{year}} conference.

Today is mostly a reminder that we're accepting talk proposals until **{{ cfp.ends }}**. Opportunity grants and Volunteer applications are still open as well. And of course, tickets are on sale!

## Submit your talk by {{ cfp.ends }}

You still have time to submit your talk proposal for the conference. Check out the [Call for Proposals page](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp) for more details about what would make a great talk.

If you already have an idea but are having a difficult time getting your proposal together, you can always ask for talk feedback on the [Write the Docs Slack](https://www.writethedocs.org/slack/). The community is here to help!

- Talk proposals are due by {{ cfp.ends }}.
- Everyone who submits will receive a response by {{ cfp.notification }}.

> ```{figure} /_static/conf/images/headers/portland-2025-speakers.jpg

Reviewer: I thought we were trying to remove the inline photos from the blog posts after it blew up last year? We should test this with mailchimp if we're going to add it. 

We can also add it to the inline page on the website with this in the header:

```
:banner: /_static/conf/images/headers/portland-2025-speakers.jpg
```

Reviewer 2: @ericholscher fixed in my commit below - images should be exactly 620px wide at 72dpi for this MailChimp template

```

## Buy your ticket for the conference

The Portland conference is {{ date.short }}. If you're excited to join us, go ahead and buy your ticket today! The [Tickets page](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/) has all the information about pricing and what is included.

Similar to our past conferences, we'll have great programming and social activities, including:

- Speaker Presentations
- Writing Day
- Unconference Sessions
- Sponsor Expo
- Sunday Welcome Reception
- Monday Night Social

The full [event website](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/) is live. We'll be updating it with information about where to stay and visiting Portland in the next couple of weeks, so stay tuned.

### Opportunity grants are open

Interested in attending but need extra financial support? We strive to provide an accessible conference for folks of all backgrounds to join. Our opportunity grant application is open until **{{ grants.ends }}**.

View our [Opportunity Grants page](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/) for information and the grant application.

### Calling all Volunteers

> Interested in being a part of making the conference happen? Apply through our [Volunteer page](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/volunteer/).

Reviewer: This could use a bit more copy as well. Perhaps a small list of role descriptions? And the fact they get a free ticket?

> ### Give feedback on Writing Day

Reviewer: I'd put this either at the top or in the intro -- feels buried here.

> We’re considering changing the format of Writing Day and want to hear your thoughts! If you’ve attended in the past, we’d love to hear what you liked, what you didn’t, and anything you think would make it better. If you haven’t attended, what kinds of things would pique your interest? Would you be more likely to attend if the sessions were shorter? Do you want sessions where you are doing actual writing, or more sessions where we discuss writing as a topic? Email [portland@writethedocs.org](mailto:portland@writethedocs.org) with any ideas.

Reviewer: ```suggestion
We’re considering changing the format of Writing Day and want to hear your thoughts! If you’ve attended in the past, we’d love to hear what you liked, what you didn’t, and anything you think would make it better. If you haven’t attended, what kinds of things would pique your interest? Would you be more likely to attend if the sessions were shorter? Do you want sessions where you are doing actual writing, or more sessions where we discuss things similar to the Unconference? Email [portland@writethedocs.org](mailto:portland@writethedocs.org) with any ideas.
```
